components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -901,6 +901,7 @@
 <!-- SHOC_SGS -->
 <do_shoc_sgs                 > .false. </do_shoc_sgs>
 <do_shoc_sgs   shoc_sgs="1"  > .true. </do_shoc_sgs>
+<shoc_timestep               > 300.0D0 </shoc_timestep>
```

Comment:
If we go to a finer timestep than 300 seconds will shoc_timestep take the new smaller value?

Reply:
Yes, it will be automatically adjusted in the SHOC interface

Reply:
great, just making sure.